samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/FAQ.md

```diff
@@ -0,0 +1,104 @@
+
+      After that initial migration, they should then run subsequent $export using "_since" and "_till".
+
+7.	Are there any configurations to be done for Destination FHIR server and how to do them?
+ 	+ For Destination FHIR server we need to configure number of RU, auto scaling, number of nodes and throttling limits, Configuration details could be found in document [here](https://github.com/Azure-Samples/azure-health-data-services-samples/blob/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/Server_%26_App_Config.md#destination-fhir-server).
+
+8.	What configurations to be considered for FHIR Bulk Loader Applications?
+	+ Users can configure App Service plan, number of instances, number of fhir resources per bundle, more details could be found [here](https://github.com/Azure-Samples/azure-health-data-services-samples/blob/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/Server_%26_App_Config.md#fhir-bulk-loader-application).
+
+9.	How do users verify the data movement?
+	+ Users can follow [Data Movement Verification](https://github.com/Azure-Samples/azure-health-data-services-samples/tree/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir#data-movement-verification) steps to verify the data import.
+
+10.	How do users troubleshoot any issue encountered during export?
+	+ Users can check the troubleshooting details from [here](https://github.com/Azure-Samples/azure-health-data-services-samples/blob/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/Error_Handling.md#export-data).
+
+11.	How do users check issues encountered with copy?
+	+ Users can check for the possible reasons for issues with copy command from [here](https://github.com/Azure-Samples/azure-health-data-services-samples/blob/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/Error_Handling.md#copying-data).
+
+12. How do users handle issues encountered during import by FHIR Loader?
+	+ There are various errors which users could encounter for FHIR Loader while importing data, the details of troubleshooting FHIR loader issue are available [here](https://github.com/Azure-Samples/azure-health-data-services-samples/blob/main/samples/fhir-to-fhir/api-for-fhir-to-api-for-fhir/docs/Error_Handling.md#fhir-bulk-loader).
+
+13.	Do we need two storage accounts for holding exported data and to be associated with FHIR loader?
+    + No users don�t need two storage accounts, Users can use storage account that gets created with FHIR Loader installation, Export the data from source Azure API for FHIR to a new container in storage account linked to FHIR Loader.
+
+14.	Can we run copy script from local machine OR it must be from Azure PowerShell?
+	+ Yes, copy script can be run from local machine for that user should install the Az CLI and AZ copy locally and log into the azure account.
+      
+       **Recommendation**: Users should run the copy script from Azure PowerShell.
```

Comment:
We should add a callout that if the user is working with private networks, this will have to be done from an execution environment that can access the storage account(s).

Reply:
changes done as per review comments.